theorems/T000092.md

```diff
@@ -7,8 +7,10 @@ then:
 refs:
 - doi: 10.1007/978-1-4612-6290-9
   name: Counterexamples in Topology
+- mathse: 4812062
+  name: Relationship between dispersion point and biconnected property
 ---
 
 Suppose $p \in X$ is a dispersion point and $X = U \cup V$ is a disjoint union then without loss of generality $p \in U$ and $V \subset X \setminus \{p\}$ is disconnected.
```

Comment:
Note that "disconnected" was never defined as such, so this is not clear and if it's means "not connected", it's even false when $V$ is a singleton or the empty set for example.  The sentence as written needs some rewording.  Maybe look at the proof in the mathse question for inspiration.

Reply:
fixed